Directory.Build.props

```diff
@@ -11,9 +11,9 @@
     <IsShipping>false</IsShipping>
 
     <ProjectDir>$(MSBuildThisFileDirectory)</ProjectDir>
-    <KeysDir>$(ProjectDir)keys/</KeysDir>
-    <SubmoduleDirectory>$(ProjectDir)src/</SubmoduleDirectory>
-    <PatchesDir>$(ProjectDir)patches/</PatchesDir>
+    <KeysDir>$(ProjectDir)src/repos/keys/</KeysDir>
+    <SubmoduleDirectory>$(ProjectDir)src/repos/src/</SubmoduleDirectory>
+    <PatchesDir>$(ProjectDir)src/repos/patches/</PatchesDir>
```

Comment:
What logic is dependent on this? I'm thinking these definitions should be relocated to a Directory.Build.props file in src/repos.

Reply:
Fixed with d703753f3d69b6448dbff4cef1f2139ccf3fc13f.